packages/client/modules/outcomeCard/components/OutcomeCardAssignMenu/TaskFooterUserAssigneeMenu.tsx

```diff
@@ -24,13 +24,13 @@ const TaskFooterUserAssigneeMenu = (props: Props) => {
   const {area, menuProps, task, viewer} = props
   const {userId, id: taskId} = task
   const {team} = viewer
-  const {teamMembers}: any = team || {teamMembers: []}
-  const taskUserIdx = useMemo(() => teamMembers.map(({userId}) => userId).indexOf(userId) + 1, [
+  if (!team) return null
```

Comment:
since this component could re-render, it may be worth it to move this check to L43 after all the hooks have run so react doesn't get a hook count mismatch across renders

Reply:
Makes sense! Strange thing with this one is [Sentry suggests](https://sentry.io/organizations/parabol/issues/2059791893/?project=107196&referrer=github_plugin) that `team` can exist with `teamMembers` being `null` even though it's [non-nullable](https://github.com/ParabolInc/parabol/blob/a9cc0145d12379e863ab6e12a74498d7cc82036c/packages/server/graphql/types/Team.ts#L277).